lib/raven/event.rb

```diff
@@ -134,6 +134,22 @@ def from_message(message, options = {})
 
       private
 
+      def get_exception_class(x)
+        x.is_a?(Module) ? x : qualified_const_get(x)
+      rescue NameError # There's no way to safely ask if a constant exist for an unknown string
+        nil
+      end
+
+      # In Ruby <2.0 const_get can't lookup "SomeModule::SomeClass" in one go
+      def qualified_const_get(x)
+        parts = x.to_s.split("::").delete_if(&:empty?)
```

Comment:
also, I'm surprised there's no Rubocop rule for this, but I would prefer `reject!` over `delete_if` here.